### PR TITLE
Mainly styles fixes and some improvements to series single and landing pages

### PR DESCRIPTION
**Pull request by rocaiguina**

### CODE REVIEW

File: wp-content/themes/cpipr/series-landing.php [hunk 1]
```diff
@@ -174,7 +171,7 @@
 								'taxonomy' => 'series',
 								'term' => $series,
 								'order' => 'DESC',
-								'posts_per_page' => $opt['per_page'],
+								'posts_per_page' => -1,
```
Comment (benlk) [inline wp-content/themes/cpipr/series-landing.php]: I can't find the link, but using `-1` to display all posts in a query is less performant than using a sufficiently large number, like `100`.

This setting should be changed through the series landing page settings, not hardcoded in the template, but if it's going to be changed here, the reason should be documented in a comment in the code, and it should still be a (large) fixed number.

File: wp-content/themes/cpipr/series-single.php [hunk 2]
```diff
@@ -156,23 +159,22 @@ function getNextPost($post, $series_map)
                     <p class="date-text"><?php the_time('j F, Y')?></p>
                   </div>
                 </div>
-              </div>
+							</div>
```
Comment (benlk) [inline wp-content/themes/cpipr/series-single.php]: What happened to the indentation on this line?

File: wp-content/themes/cpipr/series-single.php [hunk 3]
```diff
@@ -193,6 +195,7 @@ function getNextPost($post, $series_map)
 			</div>
 		</div>
 	</div>
+	<?php endif;?>
```
Comment (benlk) [inline wp-content/themes/cpipr/series-single.php]: Either this should be deindented one tab, or the opening end of the `if` statement, and all lines within the `if` statement, should be indented one tab.

File: wp-content/themes/cpipr/series-single.php [hunk 4]
```diff
@@ -134,19 +134,22 @@ function getNextPost($post, $series_map)
...
         <div class="hero-main">
           <div class="wrapper-image" style="background: url('<?php echo $heroImageNext['0']; ?>') no-repeat center/cover;">
             <div class="container-fluid">
               <div class="row-fluid">
                 <div class="span2">
-                  <h3 class="title-post">Siguiente en la serie</h3>
+									<div class="section-title">
+										<h3><?php _e('Next in Series', 'largo'); ?></h3>
+										<?php echo '<h3>' . ($next_position + 1) . ' / ' . count($series_map) . '</h3>'; ?>
+									</div>
```
Comment (benlk) [inline wp-content/themes/cpipr/series-single.php]: This section should be indented to match the surrounding code.

File: wp-content/themes/cpipr/css/style.css [hunk 5]
```diff
@@ -1535,6 +1537,9 @@ blockquote p {
   .wrapper-special .container-special > .col-xs-6 {
     width: 100%;
   }
+  .wrapper-post-serie {
```
Comment (kaylima) [inline wp-content/themes/cpipr/css/style.css:1571]: @benlk should this be `.wrapper-post-series` rather than `.wrapper-post-serie`?

**Review by benlk — CHANGES_REQUESTED** [review]
Could you make the pagination changes and indentation changes described here? I'm passing this pull request to @kaylima to review the styles.

File: wp-content/themes/cpipr/css/style.css (hunk 5 above)
Comment (benlk) [reply wp-content/themes/cpipr/css/style.css:1571]: https://github.com/INN/umbrella-cpipr/pull/27/files#diff-491875b64a837adea64560423c9819bcR155 has the class name as `wrapper-post-serie`, so this is correct.

File: wp-content/themes/cpipr/series-landing.php (hunk 1 above)
Comment (rocaiguina) [reply wp-content/themes/cpipr/series-landing.php]: ok, rolled back to original query

File: wp-content/themes/cpipr/series-single.php (hunk 2 above)
Comment (rocaiguina) [reply wp-content/themes/cpipr/series-single.php]: we are now showing the whole block only if $next_post is not empty

File: wp-content/themes/cpipr/series-single.php (hunk 3 above)
Comment (rocaiguina) [reply wp-content/themes/cpipr/series-single.php]: ok, done. please check

File: wp-content/themes/cpipr/series-single.php (hunk 4 above)
Comment (rocaiguina) [reply wp-content/themes/cpipr/series-single.php]: ok, done. please check

**Comment by rocaiguina** [comment]
@kaylima @benlk I have applied the requested changes, please let me know if we need to change anything else.

**Comment by benlk** [comment]
For some reason, I didn't know that https://make.wordpress.org/core/2016/11/03/post-type-templates-in-4-7/ was a thing. That's pretty cool. It's something that we had built out in Largo eons ago (probably 3 years ago with version 0.4), but it looks like WordPress Core added a similar functionality in 2016. Your `series-single.php` uses the WordPress Core method of changing templates, which in the post editor is controlled by the "Post Attributes/Atributos de entrada" meta box in this screenshot:

screen shot 2018-04-25 at 12 26 16 pm

If you want to set the template in the Largo way, using the "Layout Options/Opciones de emplanaje" meta box, 89bef22 is the commit that contains those changes. Making that change involves renaming the file, which will reset the layouts on your existing published posts. At some point we'll migrate the Largo way to the WordPress way, but that's not something we need to address right now.

The rest of this looks good, so I'm merging and deploying.

**Review by benlk — APPROVED** [review]

**Comment by benlk** [comment]
Now deployed.

**Pull request merged**